src/Media.js

```diff
@@ -3,11 +3,7 @@ import React from 'react';
 import elementType from 'prop-types-extra/lib/elementType';
 
 import MediaBody from './MediaBody';
-import MediaHeading from './MediaHeading';
-import MediaLeft from './MediaLeft';
-import MediaList from './MediaList';
-import MediaListItem from './MediaListItem';
-import MediaRight from './MediaRight';
+
 import { bsClass, getClassSet, splitBsProps } from './utils/bootstrapUtils';
```

Comment:
We recently added a different way to handle the styling bits, lets upgrade the Media component while we're here to use `createBootstrapComponent`, check out Button.js for a reference, we wnat to completely get rid of bootstrapUtils in each component 👍